.chloggen/jgm-pdatautil-mapkeyshash.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
What bug is being fixed?

Reply:
Help me out. Are you asking for an issue, or would `enhancement` be a better fit?

Reply:
@jacobmarble ya I'm trying to figure out if the change_type is correct.  Did the deprecation of .Sort cause a bug?

Reply:
I think it should be `enhancement` to the current functionality of this module since we are adding a new helper

Reply:
I think we should call this an enhancement. Deprecating `.Sort` is fine, I'm just adding another util function to fill in a gap in the wake of that change.

I'll update the PR.